src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -231,7 +234,9 @@
                     ContainerEnvironmentVariables="@(ContainerEnvironmentVariables)"
                     ContainerRuntimeIdentifier="$(ContainerRuntimeIdentifier)"
                     ContainerUser="$(ContainerUser)"
-                    RuntimeIdentifierGraphPath="$(RuntimeIdentifierGraphPath)"> <!-- The RID graph path is provided as a property by the SDK. -->
+                    RuntimeIdentifierGraphPath="$(RuntimeIdentifierGraphPath)"
+                    GenerateLabels="$(ContainerGenerateLabels)"
+                    GenerateDigestLabel="$(ContainerGenerateLabelsImageBaseDigest)"> <!-- The RID graph path is provided as a property by the SDK. -->
```

Comment:
Not sure I love this naming convention. Maybe `ContainerGenerateLabelForImageBaseDigest`? Or `ContainerGenerateLabel_ImageBaseDigest`? Those are admittedly not great either . . .

Reply:
I'm not enamored with it either, I was mostly just following the existing naming convention here.
The reason we even have to add these properties is because we don't resolve the user-facing base image name to the manifest digest until we actually create the image. If we were doing better MSBuild Incrementality, you could imagine a situation where we'd do

* compute desired container base image (existing target)
* fetch + resolve manifest (list) to single base image manifest from the base image (new step)
* compute container config data (existing ComputeContainerConfig target)
* download base layers (in parallel) (new step)
* create the image/push to appropriate storage (existing target, much less work done inside it)

all as separate tasks that could take advantage of MSBuild incrementality and parallelism. In this world, none of the label-generation flags would ever need to be passed to the task. So I'm viewing this as an intermediate-stage.